src/component/feature/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,101 @@
+import Button from '../../@common/Button/Button';
+import Text from '../../@common/Text/Text';
+
+import {
+  productCardButtonContainerStyle,
+  productCardContentHeaderStyle,
+  productCardContentStyle,
+  productCardImageContainerStyle,
+  productCardImageStyle,
+  productCardSoldOutStyle,
+  productCardStyle,
+} from './ProductCard.styles';
+import { CartItem, Product } from '../../../types/common';
+import { IconAddCart } from '../../../asset';
+import { useContext } from 'react';
+import CartContext from '../../../context/cartContext/cartContext';
+import CartController from '../CartController/CartController';
+import { isValidImageUrl } from '../../../util/isValidImageUrl';
+
+interface ProductCardProps extends Omit<Product, 'category'> {}
+
+const ProductCard = ({
+  id,
+  name,
+  price,
+  imageUrl,
+  quantity,
+}: ProductCardProps) => {
+  const { cartData, addCart, increaseCart, decreaseCart } =
+    useContext(CartContext);
+
+  const newImageUrl = isValidImageUrl(imageUrl)
+    ? imageUrl
+    : './image/default.jpeg';
```

Comment:
이건 리뷰어 하리한테 받았던 코멘트인데요~! 오거스는 이미 알 수도 있지만

이렇게 처리해주는 것도 충분히 좋고, 다른 방법으로는 img 태그의 onerror 를 적극적으로 활용해주는 방법도 있어요!

적용해본다면 이 부분을 삭제하고 img 태그에 바로 아래와 같이 작성해볼 수 있겠네요!😃

```suggestion
     <img css={productCardImageStyle} src={imageUrl} alt={name} onError={(e) => {e.currentTarget.src = './image/default.jpeg'; }}/>
```